lib/rspec/rails/example/system_example_group.rb

```diff
@@ -78,16 +78,19 @@ def app
         def initialize(*args, &blk)
```

Comment:
Unrelated to this pull request: wondering if it makes sense to keep `def initialize` inside `include do`. 🤔